alws/alembic/versions/63a8cf1a53ec_add_cascade_delete_to_errata_records.py

```diff
@@ -0,0 +1,143 @@
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "63a8cf1a53ec"
+down_revision = "fad009ddf89a"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.alter_column(
+        "build_releases", "owner_id", existing_type=sa.INTEGER(), nullable=True
+    )
+    op.alter_column(
+        "builds", "owner_id", existing_type=sa.INTEGER(), nullable=True
+    )
+    op.drop_constraint(
+        "errata_packages_errata_record_id_fkey",
+        "errata_packages",
+        type_="foreignkey",
+    )
+    op.create_foreign_key(
+        None,
+        "errata_packages",
+        "errata_records",
+        ["errata_record_id"],
+        ["id"],
+        ondelete="CASCADE",
+    )
+    op.drop_constraint(
+        "errata_references_cve_id_fkey",
+        "errata_references",
+        type_="foreignkey",
+    )
+    op.drop_constraint(
+        "errata_references_errata_record_id_fkey",
+        "errata_references",
+        type_="foreignkey",
+    )
+    op.create_foreign_key(
+        None,
+        "errata_references",
+        "errata_cves",
+        ["cve_id"],
+        ["id"],
+        ondelete="CASCADE",
+    )
+    op.create_foreign_key(
+        None,
+        "errata_references",
+        "errata_records",
+        ["errata_record_id"],
+        ["id"],
+        ondelete="CASCADE",
+    )
+    op.drop_constraint(
+        "errata_to_albs_packages_errata_package_id_fkey",
+        "errata_to_albs_packages",
+        type_="foreignkey",
+    )
+    op.create_foreign_key(
+        None,
+        "errata_to_albs_packages",
+        "errata_packages",
+        ["errata_package_id"],
+        ["id"],
+        ondelete="CASCADE",
+    )
+    op.alter_column(
+        "products", "owner_id", existing_type=sa.INTEGER(), nullable=True
+    )
+    op.alter_column(
+        "products", "team_id", existing_type=sa.INTEGER(), nullable=True
+    )
+    op.alter_column(
+        "teams", "owner_id", existing_type=sa.INTEGER(), nullable=True
+    )
+    # ### end Alembic commands ###
+
+
+def downgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.alter_column(
+        "teams", "owner_id", existing_type=sa.INTEGER(), nullable=False
+    )
+    op.alter_column(
+        "products", "team_id", existing_type=sa.INTEGER(), nullable=False
+    )
+    op.alter_column(
+        "products", "owner_id", existing_type=sa.INTEGER(), nullable=False
+    )
+    op.drop_constraint(None, "errata_to_albs_packages", type_="foreignkey")
+    op.create_foreign_key(
+        "errata_to_albs_packages_errata_package_id_fkey",
+        "errata_to_albs_packages",
+        "errata_packages",
+        ["errata_package_id"],
+        ["id"],
+    )
+    op.drop_constraint(None, "errata_references", type_="foreignkey")
+    op.drop_constraint(None, "errata_references", type_="foreignkey")
```

Comment:
pls add name for constraints/foreign keys in `models.py`

Reply:
fixed